packages/massa-proto-cli/src/tsProtoTypes.json

```diff
@@ -12,5 +12,20 @@
     "sfixed64": "bigint",
     "bool": "boolean",
     "string": "string",
-    "bytes": "Uint8Array"
+    "bytes": "Uint8Array",
+
+    "double[]": "number[]",
```

Comment:
are the keys `protobuf` types?
if yes, is really `type[]` a legitimate notation?

Reply:
these are not protobuf types. the [ ] allow our script to know when the expected type is an array.

I like this notation but maybe we need to use another one

Reply:
Ok, it was a naïve question :)
`[]` are very commons to denote arrays so it's perfectly fine.
An optimization would be to not duplicate every type with `[]` but, we can keep this for later.